src/backend/main.ts

```diff
@@ -79,3 +83,23 @@ function hash(input: string, pepper: string) {
     .update(input + pepper)
     .digest('base64');
 }
+
+export async function storeTotalResponsesToS3() {
+  const queryResult = await athenaExpress.query({
+    sql: 'SELECT COUNT(*) as total_responses FROM responses',
+    // TODO: Configure this
+    db: 'symptomradar_dev_storage',
+  });
+
+  // TODO: Add model for this
+  const data = queryResult.Items[0];
+
+  await s3
+    .putObject({
+      // TODO: Configure this
+      Bucket: 'symptomradar-dev-open-data',
+      Key: 'total_responses.json',
+      Body: JSON.stringify(data),
```

Comment:
If you look at the other data sets on https://data.oiretutka.fi/, they're actually pretty-printed JSON.

Not sure if someone finds that distasteful for massive data dumps, but then the JSON gzips to about 10% of its size on the wire anyway.

I would prefer to have this pretty-printed as well.